src/components/DigitalCredentials/CredentialsDashboard.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
```

Comment:
I like to see ids on components to faciliate debugging and testing. Our standard is to name it after the component itself, eg, `id="credential-dashboard"`.

Also for the other components, and any major sections/blocks/v-cards/etc.

(Use classnames if the component can be displayed multiple times, to avoid duplicate ids.)

Reply:
Looks good in all components I checked. Thanks!